test/gtest/constant_folding.cpp

```diff
@@ -0,0 +1,135 @@
+#include "frontends/common/constantFolding.h"
+#include "frontends/common/parseInput.h"
+#include "frontends/common/resolveReferences/referenceMap.h"
+#include "frontends/common/resolveReferences/resolveReferences.h"
+#include "gtest/gtest.h"
```

Comment:
```suggestion
#include <gtest/gtest.h>
```

+ clang-format on it.

Reply:
clang-format doesn't change this (at least not on my test systems).

If angled brackets are preferred, then we should change _all_ of the files in that directory to use them. Currently only `helpers.h` and `p4runtime.cpp` use angled-brackets. Let me know if this is preferred and I'll create a separate PR.

Reply:
I think it is a good idea to make all gtest includes angled, since it is a third-party dependency not in our control. It matters for some analysis frameworks. 

We can leave it as is here in this PR.

Reply:
Created a new PR for the other files: #4422